src/utils/attachArtifact.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const fs = require('fs')
+const { zip } = require('zip-a-folder')
+const github = require('@actions/github')
+const { ASSET_LABEL, ASSET_FILENAME } = require('../const')
+
+const attachArtifact = async (buildDir, releaseId, token) => {
+  const outFile = ASSET_FILENAME
+  try {
+    await zip(buildDir, outFile)
```

Comment:
have you thought about whether it's better to create a zip file or a tar archive?

Reply:
I was thinking of the zip file, but I'm happy to archive it as a tar file if needed. Do you see any benefit in doing a tar archive instead of a zip file? (probably one benefit should be that tar archives can be extracted on every filesystem without additional packages)

Reply:
no not really, let's just stick to whatever existing conventions github uses when adding artifacts to releases